addons/block_code/blocks/communication/get_node.tres

```diff
@@ -0,0 +1,27 @@
+[gd_resource type="Resource" load_steps=5 format=3 uid="uid://canpdkahokjqs"]
+
+[ext_resource type="Script" path="res://addons/block_code/code_generation/option_data.gd" id="1_bk47y"]
+[ext_resource type="Script" path="res://addons/block_code/code_generation/block_definition.gd" id="1_d60g7"]
+[ext_resource type="Script" path="res://addons/block_code/blocks/communication/get_node.gd" id="1_we5wl"]
+
+[sub_resource type="Resource" id="Resource_esr4a"]
+script = ExtResource("1_bk47y")
+selected = 0
+items = []
+
+[resource]
+script = ExtResource("1_d60g7")
+name = &"get_node"
+target_node_class = ""
+description = "Get the node at the given path"
+category = "Communication | Nodes"
+type = 3
+variant_type = 24
+display_template = "Node {path: NIL}"
```

Comment:
I would either remove the "Node" prefix from here or from the "Call method in node NODE" block, otherwise the word is repeated:
![Captura desde 2024-09-24 08-41-05](https://github.com/user-attachments/assets/f99aabf6-5d9e-4cdd-870c-838a2c568b9a)
```suggestion
display_template = "{path: NIL}"
```

Reply:
Yeah.. I think in the future we should have a small icon next to the label of value blocks indicating their type (Vector2, float, object, etc). Scratch has pointy blocks for booleans but in Godot there are more types so it will require too many shapes.